validation/stratified_couette_flow/stratified_couette_flow.jl

```diff
@@ -113,7 +125,7 @@ function simulate_stratified_couette_flow(; Nxy, Nz, arch=GPU(), h=1, U_wall=1,
     ##### Impose boundary conditions
     #####
 
-    grid = RectilinearGrid(arch, size = (Nxy, Nxy, Nz), extent = (4π*h, 2π*h, 2h))
+    grid = RectilinearGrid(arch, size = (Nxy, Nxy, Nz), extent = (4π*h, 2π*h, 2h), halo = (6, 6, 6))
```

Comment:
```suggestion
    grid = RectilinearGrid(arch; size = (Nxy, Nxy, Nz), extent = (4π*h, 2π*h, 2h), halo)
```

make this cleaner by adding `halo` as an argument to `simulate_stratified_couette_flow`.

Also, it doesn't look like the `closure` is an argument to `simulate_stratified_couette_flow`. You'll need that too. (You can also infer the halo size.)

Just as a general tip, I think you will save time if you generalize these functions first. Now, you are in the position of having to 1) walk back the chances you made and then 2) implement an interface that allows different closures. This is actually _more_ work then if you had simply done (2) to begin with.